app/Services/Gambit.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rogue\Services;
+
+use Rogue\Models\Signup;
+use RuntimeException;
+
+class Gambit
+{
+    /**
+     * The HTTP client.
+     *
+     * @var Client
+     */
+    protected $client;
+
+    /**
+     * Create a new Gambit API client.
+     */
+    public function __construct()
+    {
+        $config = config('services.gambit');
+
+        $this->client = new \GuzzleHttp\Client([
+            'base_uri' => $config['url'],
+            'auth' => [$config['user'], $config['password']],
+        ]);
+    }
+
+    /**
+     * Relay a signup event to Gambit so that we can switch
+     * the user's conversation topic.
+     *
+     * @see https://git.io/JUAiB
+     *
+     * @param Signup $signup
+     */
+    public function relaySignup(Signup $signup)
+    {
+        $payload = [
+            'userId' => $signup->northstar_id,
+            'campaignId' => $signup->campaign_id,
+        ];
+
+        if (!config('features.gambit')) {
+            info('Signup would have been sent to Gambit.', [
+                'id' => $signup->id,
+                'payload' => $payload,
+            ]);
+
+            return;
+        }
+
+        $this->client->post('/api/v2/messages?origin=signup', [
+            'json' => $payload,
+        ]);
```

Comment:
For convenience, here's the [API documentation](https://git.io/JUAiB) for this endpoint & here's [where we used to handle this in Blink](https://github.com/DoSomething/blink/blob/dfe6d5ef138bd2fb3d66a105d626f4ab6f62d6c5/src/workers/GambitCampaignSignupRelayWorker.js#L28-L47). We rely on Guzzle throwing exceptions for non-successful responses here, which will log an error to Papertrail & put this into the failed jobs table.